pallets/aleph/Cargo.toml

```diff
@@ -18,6 +18,7 @@ sp-io = { default-features = false, git = "https://github.com/Cardinal-Cryptogra
 sp-std = { default-features = false, git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.38" }
 
 pallets-support = { path = "../support", default-features = false }
+pallet-elections = { path = "../elections", default-features = false }
```

Comment:
This dependency is bad. Move the corresponding trait to primitives or best take management of  sessions out of elections to some SessionExt or sth.